autosklearn/pipeline/components/data_preprocessing/feature_type.py

```diff
@@ -237,3 +237,23 @@ def _get_hyperparameter_search_space_recursevely(
                 return FeatTypeSplit._get_hyperparameter_search_space_recursevely(
                     dataset_properties, cs, st_operation)
         return cs
+
+    def to_sklearn(self) -> sklearn.base.BaseEstimator:
+        original_column_transformer = cast(
+            sklearn.compose.ColumnTransformer,
+            self.column_transformer,
+        )
+        ct = [
+            (name, transformer.to_sklearn(), features)
+            for name, transformer, features in original_column_transformer.transformers_
+            if transformer is not None
+        ]
+        return sklearn.compose.ColumnTransformer(
+            transformers=ct,
+            # TODO potentially infer which args to copy
+            remainder=original_column_transformer.remainder,
+            sparse_threshold=original_column_transformer.sparse_threshold,
+            n_jobs=original_column_transformer.n_jobs,
+            transformer_weights=original_column_transformer.transformer_weights,
+            verbose=original_column_transformer.verbose,
+        )
```

Comment:
Why do we need to make a new one out of what already exists?

Reply:
The one that already exists contains Auto-sklearn stuff, this new one contains the same pipelines, but with Auto-sklearn stuff stripped away.